### PR TITLE
Create and style tracklist item, include in config.js

### PR DESCRIPTION
**Pull request by milenapribic**
Hi @chrisdhanaraj, can you look over tracklist item? Thanks! :wave:

### CODE REVIEW

File: client/components/TrackItem/trackitem.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  font-family: 'Helvetica Neue';
+  font-weight: 400;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.5);
+  line-height: 1.5;
+  letter-spacing: 1px;
+  color: #fff;
+
+  .song-name-style {
```
Comment (chrisdhanaraj) [inline client/components/TrackItem/trackitem.scss]: does this need to be nested?

File: package.json [hunk 2]
```diff
@@ -59,7 +59,6 @@
     "eslint-plugin-jsx-a11y": "^1.2.0",
     "eslint-plugin-react": "^5.1.1",
     "eventsource-polyfill": "^0.9.6",
-    "extract-text-webpack-plugin": "^1.0.1",
```
Comment (chrisdhanaraj) [inline package.json]: actually bring this back pls

File: package_BACKUP_4141.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "ecmaFeatures": {
+        "jsx": true
+      }
+    }
+  },
+  "devDependencies": {
+    "@kadira/storybook": "^1.27.0",
+<<<<<<< HEAD
```
Comment (chrisdhanaraj) [inline package_BACKUP_4141.json]: apparently there was some artifacts from the merge. remove this

File: package_BACKUP_4141.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    "connect-history-api-fallback": "^1.2.0",
+    "css-loader": "0.23.1",
+    "eslint": "^2.11.1",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.8.1",
+    "eslint-plugin-jsx-a11y": "^1.2.2",
+=======
```
Comment (chrisdhanaraj) [inline package_BACKUP_4141.json]: and this

File: package_BACKUP_4141.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-stage-0": "^6.5.0",
+    "css-loader": "^0.23.1",
+    "eslint": "^2.10.1",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-import-resolver-webpack": "^0.2.4",
+    "eslint-plugin-import": "^1.8.0",
+    "eslint-plugin-jsx-a11y": "^1.2.0",
+>>>>>>> development
```
Comment (chrisdhanaraj) [inline package_BACKUP_4141.json]: also this

File: package_BACKUP_4141.json (hunk 3 above)
Comment (chrisdhanaraj) [inline package_BACKUP_4141.json]: JK this is the backup. remove this file

File: client/components/TrackItem/index.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../../utils/reset.scss';
+import './trackitem.scss';
+
+const TrackItem = (props) => {
+  return (
+      <li activeClassName="is-active" className="track-item">{props.artist} <span className="song-name-style">{props.songname}</span>
```
Comment (chrisdhanaraj) [inline client/components/TrackItem/index.js]:
```
<span className="track-item__artist">{props.artist}</span>
<span className="track-item__song">{whatever}</span>
````

Comment (chrisdhanaraj) [inline client/components/TrackItem/index.js]: BEM = Block Element Modifier. MEMORIZE IT.

Comment (milenapribic) [inline client/components/TrackItem/index.js]: K!

File: client/components/TrackItem/trackitem.scss (hunk 1 above)
Comment (milenapribic) [inline client/components/TrackItem/trackitem.scss]: No!

**Comment by milenapribic** [comment]
Ok super! Made all these changes

**Comment by chrisdhanaraj** [comment]
Looks good :) merge + squash 'em in

**Pull request merged**